### ISSUE TITLE
Be able to create labels for multiple repos

### ISSUE THREAD

**Issue by shenghu** [ZenHubIO/support#1063]
Just like creating milestone for multiple repos, it is better to be able to create labels for multiple repos so that all repos can share the same set of labels.

**Comment by mhelvens** [comment]
Indeed. This only makes sense. I'm guessing contributors will agree, and that it's only a matter of prioritization?

**Comment by azenMatt** [comment]
Totally agree. We have a thread on it here - https://github.com/ZenHubIO/support/issues/597 (along with multiple-repo milestones)

One of our top priorities right now is #156 - a web and mobile view for ZenHub information. As part of this, we will be looking to make the concept of merging repositories much more user friendly - which includes better sharing of milestones and labels

**Comment by cogmind** [comment]
Global labels. I have seen a plugin for GitHub that makes it possible to export labels. A single import feature that retrieves labels and their colors would be great!

**Comment by gkgthelame** [comment]
@azenMatt You are referencing #597 and #156, but those are already closed. 
Can you give an update on multi-repo label support?

Thanks!

**Comment by joel-loewenstein** [comment]
there are multiple little tools that export the labels to json. if enabling to manage to create a label in  serveral connected repos it would be a real time saver for admin to enable an initial or update load to all repos. connected repos usually have very similar basesets of labels. either its annoying to create them manually or search for solutions and adapt them for use with GHE

**Comment by jfgossage** [comment]
I would like to see the following:

- Maintain a master list of labels and milestones accessible from the organization or user home page.
- Use this list as the base for new repositories for the organization or user.
- Add repository specific labels as required to an individual repository.
- Cut or copy and paste individual labels/milestones from one repository to another.

**Comment by tdayton** [comment]
This is important for my teams, because so far I've been unable to convince them to use a single repo for a given project.

**Comment by jasjes** [comment]
Label mismatches across repos are a major pain. If Zenhub could help us keep the list of available labels in sync across all our repos, that would be a godsend!
